vmdb/app/models/miq_provision_infra_workflow.rb

```diff
@@ -2,6 +2,7 @@ class MiqProvisionInfraWorkflow < MiqProvisionVirtWorkflow
   SUBCLASSES = %w{
     MiqProvisionRedhat
     MiqProvisionVmware
+    MiqProvisionMicrosoft
```

Comment:
Once PR #2141 is merged this will need to be changed to MiqProvisionMicrosoftWorkflow.